components/file_preview/file_preview.tsx

```diff
@@ -115,7 +109,7 @@ export default class FilePreview extends React.PureComponent {
                 <FileProgressPreview
                     key={clientId}
                     clientId={clientId}
-                    fileInfo={this.props.uploadsProgressPercent[clientId]}
+                    fileInfo={this.props.uploadsProgressPercent.get(clientId)}
```

Comment:
We shouldn't be changing the logic of how the component works with this migration.
I believe you can use the type `Record<string, FilePreviewInfo>`